commands/templates/env/manifest.yml

```diff
@@ -12,25 +12,6 @@ type: Environment
 #   vpc:
 #     id:
```

Comment:
While we are in the neighbourhood, shall we just delete the four lines above as they are more likely to cause confusion than help? If yes, applies elsewhere in too.